packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -0,0 +1,179 @@
+import React, { forwardRef, useState, useContext } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Tooltip } from '@zooniverse/react-components'
+import SVGContext from '@plugins/drawingTools/shared/SVGContext'
+
+const LineControls = forwardRef(function LineControls({
+  mark,
+  scale = 1,
+  onDelete
+},
+  ref
+) {
+  let [activePosition, setActivePosition] = useState('tr')
+  const ARROW_STROKE_COLOR = 'white'
+  const FILL_COLOR = 'black'
+  const HOVER_COLOR = 'grey'
+  const STROKE_WIDTH = 0.5 / scale
+  const OUTER_RADIUS = 40 / scale;
+  const INNER_RADIUS = 20 / scale;
+
+  const { viewBox, rotate, width, height } = useContext(SVGContext)
+  const [x, y, w, h] = viewBox.split(' ').map(n => parseInt(n, 10))
+  const point = (activePosition == 'tr')
+    ? { x: x + (50 / scale), y: y + (50 / scale) }
+    : { x: x + w - (50 / scale), y: y + (50 / scale) }
+
+  const StyledPath = styled('path')`
+    fill: ${FILL_COLOR};
+    &:hover {
+      cursor: pointer;
+      fill: ${HOVER_COLOR};
+    }
+  `
+
+  const TL = `M ${point.x - OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 1 ${point.x} ${point.y - OUTER_RADIUS}
+    L ${point.x} ${point.y - INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  const TR = `M ${point.x + OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 0 ${point.x} ${point.y - OUTER_RADIUS}
+    L ${point.x} ${point.y - INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 1 ${point.x + INNER_RADIUS} ${point.y}
+    Z`;
+
+  const BL = `M ${point.x - OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 0 ${point.x} ${point.y + OUTER_RADIUS}
+    L ${point.x} ${point.y + INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 1 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  const BR = `M ${point.x + OUTER_RADIUS} ${point.y}
+    A ${OUTER_RADIUS} ${OUTER_RADIUS} 0 0 1 ${point.x} ${point.y + OUTER_RADIUS}
+    L ${point.x} ${point.y + INNER_RADIUS}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x + INNER_RADIUS} ${point.y}
+    Z`;
+
+  const C = `M ${point.x - INNER_RADIUS} ${point.y}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x + INNER_RADIUS} ${point.y}
+    A ${INNER_RADIUS} ${INNER_RADIUS} 0 0 0 ${point.x - INNER_RADIUS} ${point.y}
+    Z`;
+
+  function move() {
+    setActivePosition((activePosition == 'tl') ? 'tr' : 'tl');
+  }
+
+  function destroy() {
+    onDelete();
+  }
+
+  function onEnterOrSpace(ev, func) {
+    if (ev.keyCode === 13 || ev.keyCode === 32) func()
+  }
+
+  return (
+    <g
+      ref={ref}
+      role='group'
+      aria-label={'Line Controls'}
+      focusable='false'
+	  transform={`rotate(${-rotate} ${width / 2} ${height / 2})`}
```

Comment:
```suggestion
    transform={`translate(${point.x}, ${point.y}) rotate(${-rotate} ${width / 2} ${height / 2})`}
```
Indentation. Also, you could add a translation here in order to make the x,y positioning simpler for the whole group.